src/components/ArticleList.js

```diff
@@ -3,9 +3,8 @@ import axios from "axios";
 import ArticleCard from "../components/ArticleCard";
 import { FlatList, View } from "react-native";
 
-const ArticleList = () => {
-  const [articleList, setArticleList] = useState([]);
-
+const ArticleList = ({ navigation }) => {
+  const [articleList, setArticleList] = useState(false);
```

Comment:
Should it be false? Why not empty array?

Reply:
can be also

Reply:
I would go with an empty array as Erik suggested

Reply:
changed!